api/catalog/api/serializers/audio_serializers.py

```diff
@@ -86,85 +87,51 @@ def validate_duration(value):
         return value.lower()
 
 
-class AudioSerializer(MediaSerializer):
+AudioHyperlinksSerializer = get_hyperlinks_serializer("audio")  # class
+
+
+class AudioSerializer(AudioHyperlinksSerializer, MediaSerializer):
     """A single audio file. Used in search results."""
 
-    fields_names = [
-        *MediaSerializer.fields_names,
-        "audio_set",
-        "genre",
-        "duration",
-        "bit_rate",
-        "sample_rate",
-        "alt_files",
-        "detail_url",
-        "related_url",
-        "category",
-    ]
-    """
-    Keep the fields names in sync with the actual fields below as this list is
-    used to generate Swagger documentation.
-    """
+    class Meta:
+        model = Audio
+        fields = [  # keep this list ordered logically
```

Comment:
What does "logically" mean here?

Reply:
By `logically` I meant an order that humans would like to read the output JSON. For example ID first, then title, then URL, foreign landing URL, more attrs, finally hyperlinks to the other endpoints etc.

Reply:
Added the sort in b107152.